### ISSUE TITLE
M117 fails

### ISSUE THREAD

**Issue by GWdd** [prusa3d/Prusa-Firmware#1443]
FW 3.5.1 for Prusa i3 Mk3

Entering the command `M117 Hello World ` does nothing to LCD display.

Trying to use the command in startup gcode and the FW parses lines above and below the M117 and ignores the display command.  

```
M109 S185 ; wait for extruder temp
M117 PINDA Soak Delay
G4 P15000 ; wait another 15 seconds to warm PINDA
```
In this case the display continues rolling the file name being printed from the SD card.

**Comment by inquisitiveKangaroo** [comment]
Hi,
Thank you for your feedback. I tried it by myself and _M117_ seems to work totally fine but there is a different problem. The _G4_ command overwrites your '_PINDA Soak Delay_' text. I wanted to exchange the _G4_ for _M860 S45_ (wait till pinda probe reach temperature 45°C), however, it also overwrite the _M117_. I understand the influence on the functionality of M117 command and we'll try to find a way how to evade it.)
Have a nice printing day.
Kangaroo

**Comment by GWdd** [comment]
Mr Inquisitor: thanks for the reply.  I suspected some interaction or msg priority to be the cause.  But others I've chatted with have experienced the issue and tried different things and all basically say the command is useless as implemented.  

In my example, since the G4 timer doesn't display anything, why would it overwrite the message I want displayed?  And oddly, the M860 Wait for PINDA command also has no display message - just a delay.  Both commands introduce an unexplained "dead period" that looks more like a crash than an intentional and needed delay.  


ps: I'm old school: the UI should have absolute priority and be instant.  I really hate the lag as I spin the control knob to navigate the UI.  I've selected the wrong thing many many times.  And I've coded UI's and know it doesn't have to be the way it is.
